extensions/amp-story-page-attachment/0.1/amp-story-draggable-drawer.js

```diff
@@ -517,8 +517,12 @@ export class DraggableDrawer extends AMP.BaseElement {
           return;
         }
         this.state = DrawerState.DRAGGING_TO_OPEN;
+        if (this.containerEl.hasAttribute('hidden')) {
```

Comment:
I think you need another toggle like this on `open` like if the shopping tag is clicked, I think it calls `open` directly and not `drag`